app/components/user/TwoFactor/TwoFactorConfigure/TwoFactorConfigureActivate/index.js

```diff
@@ -63,40 +60,17 @@ export default class TwoFactorConfigureActivate extends React.PureComponent<Prop
                   bgColor="transparent"
                   width="260"
                   height="260"
-                  className="block my4 mx-auto"
+                  className="block mx-auto"
                   level="H" // approx 30% error correction
                   style={{ maxWidth: '100%' }}
                   value={this.props.provisioningUri}
                 />
               </figure>
-            </div>
-          </Panel.Section>
-          <Panel.Section>
-            <div>
-              <strong>Secret Key</strong>
-            </div>
-            <div className="dark-gray mb1">
-              Can’t use the QR code? You can use this secret key instead.
-            </div>
-            <div className="flex">
-              <div className="input mr2 monospace" style={{ lineHeight: 1.8 }}>
-                {provisioningUriSecret}
+              <div className="dark-gray center mt2">
+                Can’t use the barcode?<br />Type in this secret key instead: <span className="monospace rounded" style={{ fontSize: 13 }}>{provisioningUriSecret}</span>
```

Comment:
ah, I thought this had to be the whole URI and not just the secret?

Reply:
So did I originally, turns out I was telling lies!